common/src/main/java/com/netflix/conductor/common/metadata/workflow/WorkflowDef.java

```diff
@@ -295,6 +298,15 @@ public void setVariables(Map<String, Object> variables) {
         this.variables = variables;
     }
 
+    public Map<String, Object> getInputTemplate() {
+        return inputTemplate;
+    }
+
+    public WorkflowDef setInputTemplate(Map<String, Object> inputTemplate) {
```

Comment:
To keep it consistent with the other setters in the class, can this also return `void`?

Reply:
Will do!